conf/messages.en

```diff
@@ -349,6 +349,48 @@ trusteesInfo.5mld.details.paragraph2 = This is usually where the business is reg
 
 trusteesInfo.5mld.warning = All trustees have the same legal responsibilities.
 
+trustee.individual.5mld.countryOfNationalityYesNo.title = Do you know the trustee’s country of nationality?
+trustee.individual.5mld.countryOfNationalityYesNo.heading = Do you know {0}’s country of nationality?
+trustee.individual.5mld.countryOfNationalityYesNo.checkYourAnswersLabel = Do you know {0}’s country of nationality?
+trustee.individual.5mld.countryOfNationalityYesNo.error.required = Enter yes if you know the trustee's country of nationality
+trustee.individual.5mld.countryOfNationalityYesNo.hint = The country of nationality is usually where the trustee was born and where their passport is registered.
+
+trustee.individual.5mld.countryOfNationalityInTheUkYesNo.title = Does the trustee have UK nationality?
+trustee.individual.5mld.countryOfNationalityInTheUkYesNo.heading = Does {0} have UK nationality?
+trustee.individual.5mld.countryOfNationalityInTheUkYesNo.checkYourAnswersLabel = Does {0} have UK nationality?
+trustee.individual.5mld.countryOfNationalityInTheUkYesNo.error.required = Enter yes if the trustee has UK nationality
+
+trustee.individual.5mld.countryOfNationality.title = What is the trustee’s country of nationality?
+trustee.individual.5mld.countryOfNationality.heading = What is {0}’s country of nationality?
+trustee.individual.5mld.countryOfNationality.checkYourAnswersLabel = What is {0}’s country of nationality?
+trustee.individual.5mld.countryOfNationality.error.required = Enter a country
+trustee.individual.5mld.countryOfNationality.error.length = The country of nationality must be 100 characters or less
+trustee.individual.5mld.countryOfNationality.error.invalidCharacters = The country of nationality must only include letters a to z, apostrophes, commas, full stops, hyphens, round brackets and spaces
+
+trustee.individual.5mld.countryOfResidenceYesNo.title = Do you know the trustee’s country of residence?
+trustee.individual.5mld.countryOfResidenceYesNo.heading = Do you know {0}’s country of residence?
+trustee.individual.5mld.countryOfResidenceYesNo.checkYourAnswersLabel = Do you know {0}’s country of residence?
+trustee.individual.5mld.countryOfResidenceYesNo.error.required = Select yes if you know the trustee’s country of residence
+trustee.individual.5mld.countryOfResidenceYesNo.hint = The country of residency is usually the country where the trustee lives and works most of the time during the tax year.
+
+trustee.individual.5mld.countryOfResidenceInTheUkYesNo.title = Is the trustee a UK residence?
```

Comment:
`resident` instead of `residence`?

Reply:
Is this still outstanding?

Reply:
This should be done now. I missed this one when going through the comments